component-samples/demo/aio/new-device.sql

```diff
@@ -12,8 +12,9 @@
 
```

Comment:
@DukeDavis12 @trbehera , Do we also want to change the default svi bat file and payload or we just do this for Linux alone?

Reply:
We thought about changing only the Linux version so that when we setup automation against client-sdk, we could have same rule while validating.